models/model.js

```diff
@@ -6,7 +6,15 @@ exports.fetchTopics = () => {
         `SELECT * FROM topics;`
       )
       .then((result) => {
-        // console.log(result.rows);
         return result.rows;
       })
+};
+
+
+exports.selectArticleById = (article_id) => {
```

Comment:
This is where (in the models) you want to check if your article_id is valid, or exists in your database.

Reply:
Fixed